test/assembly/BUILD

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2020 Grakn Labs
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+load("@graknlabs_build_tools//checkstyle:rules.bzl", "checkstyle_test")
+
+java_test(
+    name = "daemon-test-assembly",
+    srcs = [
+        "AssemblyConstants.java",
+        "DaemonTest.java",
+    ],
+    classpath_resources = ["//test/resources:logback-test"],
+    data = ["//:assemble-mac-zip"],
+    test_class = "grakn.core.assembly.DaemonTest",
+    deps = [
+        "//common",
+        "//server",
+        "//dependencies/maven/artifacts/commons-io",
+        "//dependencies/maven/artifacts/org/hamcrest:hamcrest-library",
+        "//dependencies/maven/artifacts/org/zeroturnaround:zt-exec",
+        "@graknlabs_client_java//:client-java",
+    ],
+)
+
+java_test(
+    name = "distribution-test-assembly",
```

Comment:
Let's reverse the naming, and have `test` be the prefix. So this one we can call `test-distribution`